docs/sources/flow/reference/components/loki.write.md

```diff
@@ -80,6 +80,7 @@ Name                  | Type          | Description                           |
 `proxy_url`           | `string`      | HTTP proxy to proxy requests through. | | no
 `follow_redirects`    | `bool`        | Whether redirects returned by the server should be followed. | `true` | no
 `enable_http2`        | `bool`        | Whether HTTP2 is supported for requests. | `true` | no
+`retry_on_http_429`   | `bool`        | Retry when an HTTP 429 status code is received. | `true` | no
```

Comment:
There is a comment in `prometheus.remote_write` which I think we should port:

> The retry_on_http_429 argument specifies whether HTTP 429 status code responses should be treated as recoverable errors; other HTTP 4xx status code responses are never considered recoverable errors. When retry_on_http_429 is enabled, Retry-After response headers from the servers are honored.

Can you also check if this is accurate with regards to `Retry-After` header comment? Want to make sure we're documenting the actual behaviour.

Reply:
Unless Im missing something, it seems that the code is not using the Retry-After header value in this case. It is using the backoff parameters:
`		level.Warn(c.logger).Log("msg", "error sending batch, will retry", "status", status, "tenant", tenantID, "error", err)
		c.metrics.batchRetries.WithLabelValues(c.cfg.URL.Host, tenantID).Inc()
		backoff.Wait()
`
The backoff parameters can be configured by the users with MinBackoff, MaxBackoff and MaxRetries.

Reply:
Oh, I see. Thanks for checking! Indeed, I can see that we use backoff config. And prometheus.remote_write uses "Retry-After" header. 

We should adapt the comment and still add it to docs.